distribution/lib/Standard/Database/0.0.0-dev/src/Data/Column.enso

```diff
@@ -1103,7 +1147,8 @@ type Column
         new_name = self.naming_helper.function_name "ends_with" [self, other]
         make_text_case_op self "ends_with" other case_sensitivity new_name
 
-    ## Checks for each element of the column if it contains `other`.
+    ## GROUP Standard.Base.Logical
+       Checks for each element of the column if it contains `other`.
```

Comment:
Inconsistency.

I'm pretty sure that `starts_with`, `ends_with` and `contains` should belong to the same group.	But this one is in `Logical` whereas `ends_with` is in `Text`. IMO all of these belong to `GROUP Text`.